pantheon/src/main/java/tech/pegasys/pantheon/cli/PantheonCommand.java

```diff
@@ -658,6 +678,19 @@ public void run() {
     }
   }
 
+  private void checkFastSyncOptions() throws ParameterException {
+    if (fastSyncMaxWaitTime < 0) {
```

Comment:
Our parameter validation has been more just in time and directly in the configuration options rather than directly between the CLI and the config.  It keeps the validations closer to where the parameters are actually stored.  Validating in the CLI also provides a barrier to splitting the code apart.

Reply:
A custom converter to restrict the integer to a positive non null one ?

Reply:
None of other integers have restriction. For example metrics interval in seconds are not checked too. So what is the best practice to perform those checks ?

Reply:
I suggest we keep like this for this PR. We need to define a way to do this kind of checks and implement it for all options in a separate ticket imo.